.travis.yml

```diff
@@ -0,0 +1,21 @@
+services:
+- docker
+env:
+  global:
+  - secure: iXLBgM1X3DgWPQzZmvLfoacNW9cykid48FF1gvC4Sah5mD8xQv/zUDWgxi63448C+jSPD6pksJ3UUEAOFL1V7kWUbVDvUK3r8WdNQHE1oJvExZduXm7oazQgov0CFMdU+P59GKgBa4WfGNy6c9lNyaI6AU/bAm2e9Gm6w0DAcn7P7upltAYmBIP3KSTl9m4qVj6ZP6JI/W2h/LWmi/a/CZLVRdFtOjpmynonoj49KvsyPhAvjiIh3c1nJx5kSaaxx9ThULT/y8IApVzZO7OQ3mJ6gbu6X9JpJSf2GQ0Z0ghdkBzWEquJDDIV5Joqo6dXkGC1QmJ9PcGwDtHMSMNlmt8H+YyZnfGaDzZFQnZh9mgH3h9nFgKANk+Yhb2ipgti+qnS0sSnOJpq6ytwX8Lhl/33XoP5ED3FP0435UTNK1OujSUQ0u7hthw3BnZCOXyP1R7uK+7U3usznF7CkhmFwl0Rnin5l9IMfyGJzq2q3IxpiZ6z2ZCz7/maVKYNGLSqpBxzbTo+7eFbvxQKpD1o/Zf4O17wMo0oF3e0OxR3ISj6kphQDbaO/q4R6+4D40iJgYFPmgRjVazZcP2Zueu0XzQiBaRRTMp5nOuq0YpTZOdVMqzIQoXl09vA/9Zxkpv6iw3pktwNy4Tqh9s8iNhx2GplVlzQfjsEC6xOReVCfs4=
+  - secure: IuLuPb0QJ+FkdJtbLIYiIDREfYgvpCrOuZIeYPMLAvqJ/Zm16n7+JmreNm0VDdtp3pLJz/RS/d1W6gUJLm1+C2U+fCGnIbQSZ06d7zldZJw3g8O7UwtGUXfGf6oo5jsgluDhVkdJjYqFA9592vX9b0L9HAQkUyNCub6ofrdaCv3DF90Fe/i12csCkOTjmpFaDq8lIORyWbWSZpt4V97os72I44y4SN4IZexhinSS8LsGWCnP1ttdXsKiI0qK/ICWXi8hpjoOOswpDdIssqWtsRStmKBqZnxML2k18JQytintqiQPMu+FmsU5zPxoPqQzppkR3i9JIzVQm+3LjgofYIc7I4kOc8VPgURp4gn2MbRbJt7sThtsALaN2Do/ppFBTtIt0Kbs/OXOagtZkfeOK4mhFL8ECErM3q40xqfd0qVNrkddUTN6dOJZgh9j2S88QdN1W6FYYrFgpAgYm/y4MsOSPpzVK3R5G0t7UQ2uMmOfhLZolmBPMtBVJLGsv+vkdU0d2F0kajes6kRjoB9+ZhtUmuL/HZygrRdoeI6NItJl6Hk2KHmse4yLdAPpUaDLIJFz3hLWj0MR7OHQOZ82zoiMnXR+KpUK5MnGFtU7IOG3kmShryBIRBJg5ubOywZuh2Bk4vOY3P/TDQhC1kPAc4HrrrcmCtcY0SnwFFV51OY=
+  - secure: mEpqWg5ctylIcZbFwvg2UZsMcKdj4m8qMfCFR2buinZMPV+6NPxnNOHGI6CNhCr0g2+ArdGt5giseXyojuJe0gA+K8XRo8SIFTbolh7+1uwbMDCBYhRcm46ubqRKFqWG9vqaV08lSb73uBnB7iOdmPrYFObN7njwBPHDkb0wnWlhLRybqHjr1lYh5GrZGyqmWrq/320KVt7pLc8Vy7Z9FMSfh/TwQYKx0Ak+YVm0NOpvD/GhmkyzPW6RzA2xQfxR23NF6ohxX/8bjYrJAQfvNDEO+jyZn5VG0KuaDqCJENuBtk+CJVLwI8woNxClWTHioh5fqaUAQq6v8kj6HB4UoYeGiOyBcaVngbNr1e93dKiSotYjc2yWRhikuvjug2bCGt91aeIuaeTnlbvlWH7u0B1k9FLTG/sRDXyBJO0AbTWtr+Cpao5F7g8Fw1TARmaSY+SqUGAnu6/9zaJzlrKGIX/UiJGhQQ0BTQKO4smW5J5w1bBFyFpdFAzJ3wptTSFunBvvoL8UJxsi/I2ZD54KMLjPb22nT/SgTMoHthzybZrpE9BAWKhavP7phSfHCSleHha/hhDo2Dn1+lPPQMkyfjQso6TOSL1VT2mDqY2uWYd33ox/WCdsGo+a8eENoEZ5dLDZDDpkqvsE4Od/V/iKL3vcvWCRRhliPe+E0KqvFEc=
+script:
+- docker build --target test -t todo-app-test:test .
+- docker run --env-file .env.test todo-app-test:test test/test_view_model.py
+- docker run --env-file .env.test todo-app-test:test test/test_client.py
+- docker run -e TRELLO_KEY=$TRELLO_KEY -e TOKEN=$TOKEN -e SECRET_KEY=$SECRET_KEY todo-app-test:test
+  test/test_app.py
```

Comment:
The point of having two test folders is you can just run two commands:
`docker run --env-file .env.test todo-app-test:test test`
`docker run -e TRELLO_KEY -e TOKEN todo-app-test:test test_e2e`

Also note the shorthand for the `-e` options and also SECRET_KEY shouldn't be required - it was only necessary for using flask's session cookie. If you aren't using `session`, can remove any reference to SECRET_KEY, e.g. from flask_config.py

Reply:
I'm unable to run tests if they are not from tests folder. I keep getting:
![image](https://user-images.githubusercontent.com/21199259/117008418-9c4fe480-ace2-11eb-98b5-f3957d42d2b6.png)